src/hooks/useDisputableVotes.js

```diff
@@ -133,16 +136,40 @@ async function processVote(vote, apps) {
     naysPct: vote.naysPct,
     yeasPct: vote.yeasPct,
     status: vote.status,
+    target: getTargetData(apps, disputableVotingApp.address),
   }
 
   if (vote.script === EMPTY_SCRIPT) {
     return extendedVote
   }
 
   const description = await describeScript(vote.script, apps)
+  if (description[0] && description[0].to) {
```

Comment:
Should we destructure / assign this on a previous line to save us accessing via the index each time?